### ISSUE TITLE
Navbar with fixed header/drawer breaks at certain screen widths

### ISSUE THREAD

**Issue by truenorth** [google/material-design-lite#1218]
To reproduce:
1. Set browser window width to 1000px
2. Visit http://www.getmdl.io/components/index.html#layout-section/layout
3. Find the "Fixed header and drawer" example

![screen shot 2015-07-26 at 7 07 17 pm](https://cloud.githubusercontent.com/assets/558005/8897412/b2b0d11e-33c9-11e5-9ed1-0f8efaf75d39.png)

Root cause may be related to the `mdl-layout__header`

![screen shot 2015-07-26 at 7 09 10 pm](https://cloud.githubusercontent.com/assets/558005/8897418/f056ece2-33c9-11e5-8aec-454f271ce42f.png)

https://github.com/google/material-design-lite/blob/master/src/layout/_layout.scss#L241-L244
```scss
    .mdl-layout--fixed-drawer:not(.is-small-screen) > & {
      margin-left: $layout-drawer-width;
      width: calc(100% - #{$layout-drawer-width});
    }
```

The drawer slides out at a screen width of 1025px, and the `.is-small-screen` class is added at  < 850px. Between 851px and 1024px, things look strange

**Comment by mspl13** [comment]
Duplicate of #1185 and #1197 and fixed (but not merged yet) in #1186.

**Comment by addyosmani** [comment]
Closing as it should now be fixed. Thanks for the comment, @mspl13!
